src/payment/data/utils.js

```diff
@@ -273,3 +274,17 @@ export const chainReducers = (reducers) => {
     );
   };
 };
+
+/**
+ * Create a Routine with Custom Steps
+ * @param {string} name Name of the Routine
+ * @param {string[]} addtlStages An Array of Additional Steps (these will be uppercased)
+ * @param {boolean=} keepDefaults If we should include the normal Routine Steps
+ * @returns {*} A Redux Saga Routine
+ */
+export function createCustomRoutine(name, addtlStages, keepDefaults = true) {
```

Comment:
Wraps the odd pattern of how Routines permit custom ones. Basically when you invoke `createRoutineCreator` it is a higher order function that then performs the generation when you name it after...

For example:
`export const fetchCaptureKey = createRoutine('FETCH_CAPTURE_KEY');`
is the same as 
`export const fetchCaptureKey = createRoutineCreator(defaultRoutineStages)('FETCH_CAPTURE_KEY');`

Which almost reads backwards, and someone would have to know the underlying details as to why. This simplifies that instead of writing:
`export const pollPaymentState = createRoutineCreator([...defaultRoutineStages, 'RECEIVED'])('UPDATE_PAYMENT_STATE');`
We now just write:
`export const pollPaymentState = createCustomRoutine('UPDATE_PAYMENT_STATE', ['RECEIVED']);`